Source/MonoGame.Extended/Maps/Tiled/TiledMap.cs

```diff
@@ -41,6 +40,7 @@ public void Dispose()
         public int Height { get; }
         public int TileWidth { get; }
         public int TileHeight { get; }
+        public List<TiledTileset> Tilesets { get; }
```

Comment:
It's not ideal that we are exposing mutable `List` properties. In this case it's probably okay but most of the other collection properties are exposed as `IEnumerable`'s to prevent adding and removing elements after loading. 

The `ObjectGroups` property should have never been exposed as a `List`. Now it's a breaking change to go back the other way unfortunately.

Now that we're on .NET 4.5 it might be better to change them to `IReadOnlyCollection` instead. Although, with a little redesign we could get rid of the `Create` method pattern entirely and expose our own custom collections that allow *normal* `Add` and `Remove` operations.

Anyway, starting to rant.